mcr.cpp

```diff
@@ -1,98 +1,55 @@
 #include <iostream> 
 
-using namespace std; 
+using namespace std;
 
 bool isWin(char game[3][3]){ 
-
```

Comment:
format the code, in order to be easier understanded.

Reply:
Sir, yes sir.